### PR TITLE
Circleci project setup

### PR DESCRIPTION
**Pull request by n-iwamoto**

### CODE REVIEW

**Review by dmatsubayashi — APPROVED** [review]
Not sure I'm qualified to review this but looks good to me!

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+# See: https://circleci.com/docs/configuration-reference
+version: 2.1
+
+orbs:
+  slack: circleci/slack@2.5.0
```
Comment (quentin-dietz) [inline .circleci/config.yml]: I think this slack orb can be removed

**Review by quentin-dietz — APPROVED** [review]
beautiful! nice documentation too :)

File: .circleci/config.yml (hunk 1 above)
Comment (n-iwamoto) [reply .circleci/config.yml]: Poof!

**Pull request merged**